truffle-config.js

```diff
@@ -27,8 +27,9 @@ module.exports = {
   mocha: {
     reporter: 'eth-gas-reporter',
     reporterOptions: {
+        coinmarketcap: process.env.COINMARKETCAP_API_KEY,
         currency: "USD",
-        gasPrice: 50,
+        gasPrice: 100,
```

Comment:
This `gasPrice` is set to 100 just to be the same as the one used in the `witnet-ethereum-block-relay` repository